nav-app/src/app/layer-upgrade/layer-upgrade.component.html

```diff
@@ -0,0 +1,191 @@
+                                    <changelog-cell
+                                        [tactic]="tactic"
+                                        [technique]="getTechnique(attackID, compareTo)" 
+                                        [viewModel]="compareTo"
+                                        [isCurrentVersion]="false">
+                                    </changelog-cell>
+                                </td>
+                                <td class="narrow button-container" *ngIf="section !== 'additions' && section !== 'deprecations'">
+                                    <!-- copy annotations/revert copy -->
+                                    <button class="button" matTooltipPosition="above" matTooltip="copy annotations"
+                                        [disabled]="isCopied(getTechnique(attackID, compareTo), tactic) || !anyAnnotated(attackID)"
+                                        (click)="onCopy(attackID, tactic)"><mat-icon inline="true">chevron_right</mat-icon></button>
+                                    <button class="button" matTooltipPosition="above" matTooltip="revert copy"
+                                        [disabled]="!isCopied(getTechnique(attackID, compareTo), tactic)"
+                                        (click)="onRevertCopy(attackID, tactic)"><mat-icon inline="true">chevron_left</mat-icon></button>
+                                </td>
+                                <td *ngIf="section !== 'deprecations'">
+                                    <!-- technique from latest version-->
+                                    <changelog-cell
+                                        [tactic]="tactic"
+                                        [technique]="getTechnique(attackID, viewModel, section)" 
+                                        [viewModel]="viewModel"
+                                        [isCurrentVersion]="true">
+                                    </changelog-cell>
+                                </td>
+                            </tr>
+                        </table>
+                    </div>
+
+                    <!-- reviewed -->
+                    <div class="checkbox right">
+                        <input id="review_{{attackID}}" class="checkbox-custom" type="checkbox"
+                            [checked]="isReviewed(attackID)" (click)="reviewedChanged(attackID)">
+                        <label for="review_{{attackID}}" class="checkbox-custom-label noselect">reviewed</label>
+                    </div>
+
+                    </ng-template> <!-- end technique lazy render -->
+
+                </mat-expansion-panel>
+            </ng-container>
+
+            </ng-template> <!-- end changelog lazy render -->
+
+        </mat-expansion-panel>
+        </mat-accordion>
+    </div>
+
+    <mat-divider></mat-divider>
+
+    <!-- done button -->
+    <div class="done-button">
+        <span>Updated {{changelog.reviewed.size}}/{{changelog.length()}} techniques</span>
```

Comment:
Should the total count here take into account the "show annotated techniques only" count? Otherwise it implies a lot of work has been skipped for users who only want to adjust existing annotations and not review literally every change type for unannotated techniques.